ingestion-beam/src/main/java/com/mozilla/telemetry/aet/DecryptAetIdentifiers.java

```diff
@@ -69,11 +63,6 @@
   private transient Schema telemetryAetSchema;
   private transient Schema structuredAetSchema;
 
-  private static final Pattern TELEMETRY_URI_PATTERN = Pattern
```

Comment:
These regex patterns go away because we now have ParseUri before this transform is executed.